### PR TITLE
Document scripts section

### PR DESCRIPTION
**Pull request by hverlin**
- document `scripts` section
- Fix `mdbook` task (seems that the default theme was slightly broken?)

image

### CODE REVIEW

File: docs/book.toml [hunk 1]
```diff
@@ -16,11 +16,11 @@ create-missing = false
 [preprocessor.links]
 
 [preprocessor.catppuccin]
-assets_version = "0.1.1" # DO NOT EDIT: Managed by `mdbook-catppuccin install`
+assets_version = "2.0.1" # DO NOT EDIT: Managed by `mdbook-catppuccin install`
```
Comment (hverlin) [inline docs/book.toml:19]: When running `cargo make update-book`, got the following error:

```
[cargo-make] INFO - Running Task: update-book
2023-10-14 10:25:04 [INFO] (mdbook::book): Book building has started
[2023-10-14T08:25:04Z ERROR mdbook_catppuccin] Outdated asset version '0.1.1' found:
    Please update your version of 'mdbook-catppuccin' to '2.0.1'
    Then run 'mdbook-catppuccin install' to install the latest assets.
[2023-10-14T08:25:04Z WARN  mdbook_catppuccin] Difference in Major Version Detected.
    Please see the version compatibility table to understand if you need to update your 'index.hbs'
    -> https://github.com/catppuccin/mdBook?tab=readme-ov-file#version-compatibility <-
    You can overwrite the 'index.hbs' by running `mdbook-catppuccin install --force'
[2023-10-14T08:25:04Z ERROR mdbook_admonish] Fatal error: ERROR:
      Incompatible assets installed: required mdbook-admonish assets version '^3.0.0', but found '2.0.0'.
      Please run `mdbook-admonish install` to update installed assets.
      For more information, see: https://github.com/tommilligan/mdbook-admonish#semantic-versioning
[2023-10-14T08:25:04Z ERROR mdbook_admonish]   - ERROR:
      Incompatible assets installed: required mdbook-admonish assets version '^3.0.0', but found '2.0.0'.
      Please run `mdbook-admonish install` to update installed assets.
      For more information, see: https://github.com/tommilligan/mdbook-admonish#semantic-versioning
2023-10-14 10:25:04 [ERROR] (mdbook::utils): Error: The "admonish" preprocessor exited unsuccessfully with exit status: 1 status
[cargo-make] ERROR - Error while executing command, exit code: 101
[cargo-make] WARN - Build Failed.
```

**Comment by hverlin** [comment]
Going to use https://copier.readthedocs.io/ instead, so closing this PR. Please don't hesitate to re-use the code if needed

**Comment by davidB** [comment]
Thanks for the PR. I'll take a look.
I hope `copier` will match your needs.

Why did you give `ffizer` a try before switching, as you seem to know already `copier` (regarding one of the comments in another ticket)?

**Comment by hverlin** [comment]
- `copier` is a lot more annoying to install than `ffizer` since it requires a working python environment or to package it with docker. 
- I also felt like Handlebars might be a better choice than jinja2, as support in other languages is much better

**Pull request closed without merging**